client/my-sites/themes/themes-toolbar-group/index.tsx

```diff
@@ -14,14 +14,11 @@ const ThemesToolbarGroup: React.FC< ThemesToolbarGroupProps > = ( {
 	selectedKey,
 	onSelect,
 } ) => {
-	const activeIndex = useMemo(
-		() =>
-			Math.max(
-				items.findIndex( ( { key } ) => key === selectedKey ),
-				0
```

Comment:
When the filter tab doesn't have a match, it should not display any active tab, instead of defaulting to an active index of 0.